pkg/storage/errors.go

```diff
@@ -8,17 +8,29 @@ import (
 	"github.com/openfga/openfga/pkg/tuple"
 )
 
-// since these errors are allocated at init time, it is better to leave them as normal errors instead of
-// errors that have stack encoded.
 var (
-	ErrCollision                = errors.New("item already exists")
+	// Creation errors
+
+	ErrCollision = errors.New("item already exists")
```

Comment:
🤔 To aid with godoc documentation generation, what do you think about following the best practices presented here: https://go.dev/doc/comment ?

So we'd have: 

```suggestion
        // ErrCollision is thrown when an item already exists within the store.
	ErrCollision = errors.New("item already exists")
```

This can be later down the line enforced with linters such as [revive](https://golangci-lint.run/usage/linters/#revive).